src/selectors/categorySelector.js

```diff
@@ -0,0 +1,10 @@
+import { createSelector } from 'reselect';
+
+/**
+ * @param {*} state - The initial state from the redux store
+ */
+const categorySelector = state => state.articlesCategoryReducer.category;
+const categoryIsLoadingSelector = state => state.articlesReducer.loading;
```

Comment:
```suggestion
const categoryIsLoadingSelector = state => state.articles.loading;
```
Similar to the suggestion above.